src/local.jl

```diff
@@ -59,7 +54,7 @@ function localzone()
 
             if startswith(name, '/')
                 return @mock open(name) do f
-                    read_tzfile(f, "local")
+                    read_tzfile(f, something(_path_tz_name(name, mask), "local"))
```

Comment:
This is meant to error if it can't construct the path right?

Reply:
If the path specified by the TZ environment variable doesn't exist or it isn't a tzfile an error will be raised which is expected. We could possibly raise a custom error message or add a comment indicating that.

Reply:
I think the default julia error if `nothing` is returned is probably enough information

Reply:
Oh, I think I misunderstood what you meant. The `something` here is just there to fall back to "local" if `_path_tz_name` returns `nothing`:
```julia
julia> something(nothing, "local")
"local"
```
Essentially we want to use a valid time zone name if we can find one but if this is some crazy custom time zone we'll just name it "local"

Reply:
I think looking that the commit where this change was made should clear things up: https://github.com/JuliaTime/TimeZones.jl/pull/258/commits/0e5b8e346736131fd706193be31ea025ef4d8c8f

Reply:
Oh that makes sense, reading is hard for me sometimes :)